firmware/applications/SignBuddy/nucleo_board.h

```diff
@@ -20,7 +20,8 @@
 #define SYSCLK_FREQ                     24000000
 
 #define SYSTEM_TIME_PRIORITY            4
```

Comment:
we need to fix the interrupt priorities - there are only 4 priorities on a cortex m0. 

that's my bad